tests/checkpoints_test.py

```diff
@@ -69,4 +71,4 @@ def test_get_checkpoint_handler_pytorch(no_env_var):
     """Check that checkpoint_handler type is correct for when checkpoint_handler name resolves to pytorch"""
```

Comment:
This is the reason for the `pytest.importorskip` right? For a future issue we should make a `pickled_dict_checkpoint_test.py` with similar tests to what you wrote for tf and move this test into there.

Reply:
Yep, added a TODO